vignettes/RPyGeo.Rmd

```diff
@@ -18,13 +18,19 @@ knitr::opts_chunk$set(
 
 ## Introduction
 
-RPyGeo is a package providing access to ArcGIS geoprocessing tools in R. It archives this by building an interface between R and the ArcPy Python side-package via the reticulate package. 
-
-RPyGeo is not just a wrapper package for ArcPy. It aims to create an environment, in which the user can perform ArcGIS and R functions on the same dataset in a contiguous workflow. It should not be necessary to touch Python or ArcGIS. 
+**RPyGeo** establishes an interface to the geoprocessing tools of ArcMap in R.
+Since ArcMap only provides a Python API for a low-level access of its geoalgorithms, **RPyGeo** establishes a tunnel to Python via the **reticulate** package.
+<!-- what do you want to say with the next three sentences? -->
+RPyGeo is not just a wrapper package for ArcPy.
+It aims to create an environment, in which the user can perform ArcGIS and R functions on the same dataset in a contiguous workflow. 
+It should not be necessary to touch Python or ArcGIS. 
 
 ## ArcPy
 
-ArcPy is a Python side-package that provides the user with Python access to almost any geoprocessing tools in ArcGIS. ArcPy is organized into modules, which are Python files with functions and classes. The ArcPy main module has several hundred functions and classes. It is supported by various modules like the `data access` module or extensions like the `spatial analysis` extension, which increase the number of functions and classes further. 
+<!-- again not really sure what you want to say with the subsequent paragraph, maybe we should talk in person-->
+ArcPy is a Python side-package that provides the user with Python access to almost any geoprocessing tools in ArcGIS <!-- source! -->. 
+ArcPy is organized into modules, which are Python files with functions and classes. The ArcPy main module has several hundred functions and classes. 
+It is supported by various modules like the `data access` module or extensions like the `spatial analysis` extension, which increase the number of functions and classes further. 
 
```

Comment:
I want to give a short overview of ArcPy. People who just use ArcMap do not know the essential vocabulary like module, function and class. Later on it's important to know how ArcPy is organized. Otherwise you do not know how to find the functions in RPyGeo because the functions are nested by their module e.g. `arcpy$sa$Slope`

Reply:
Ok, this is a good idea but I would keep it as short as possible and reference the corresponding ArcGIS help pages for more detailed information. I will rewrite accordingly and you then you can revise or add something to my suggestion.